### ISSUE TITLE
EVA report lost in Nirvana when vessels's HDD is full and no DNS connection available ...

### ISSUE THREAD

**Issue by Gordon-Dry** [Kerbalism/Kerbalism#673]
... in the moment the Kerbals boards the vessel.

**Comment by UsernamesAreAlwaysTaken** [comment]
I don't see that as a problem? No storage = can't store experiments. More hard-drives or a connection to manage the data?

Looks like intended functionality.

**Comment by lordcirth** [comment]
> I don't see that as a problem? No storage = can't store experiments. More hard-drives or a connection to manage the data?
> 
> Looks like intended functionality.

In stock, if you try to enter a craft that can't store your science, there's a warning & prompt to either cancel or destroy the data.

**Comment by UsernamesAreAlwaysTaken** [comment]
Fair enough. I still think the mechanic is working as intended, it probably misses a warning.

**Comment by gotmachine** [comment]
There is a stock-alike popup implemented in Kerbalism 4. 
Could be backported if someone is willing to do it : https://github.com/Kerbalism/Kerbalism/blob/b55459a96c95ac7d51b0058b279e32f688e22baa/src/Kerbalism/System/Callbacks.cs#L335-L379
